string edge_num
float64 curr_CPU
float64 curr_mem
float64 curr_thpt
string[] current_nodes